app/views/staff/events/show.html.haml

```diff
@@ -1,158 +1,227 @@
+            -# %li.list-group-item
+            -#   %strong.text-primary Confirmed:
+            -#   %span.label.label-info #{event.proposals.accepted.confirmed.count} (#{event.confirmed_percent})
             %li.list-group-item
-              %strong.text-primary Confirmed:
-              %span.label.label-info #{event.proposals.accepted.confirmed.count} (#{event.confirmed_percent})
-            %li.list-group-item
-              %strong.text-primary Scheduled:
+              %strong.text-primary Accepted Sessions Scheduled:
               %span.label.label-info #{event.proposals.scheduled.count} (#{event.scheduled_percent})
             %li.list-group-item
-              %strong.text-primary Waitlisted:
+              %strong.text-primary Waitlisted Sessions:
               %span.label.label-info #{event.proposals.waitlisted.count}
-            %li.list-group-item
-              %strong.text-primary Confirmed:
-              %span.label.label-info #{event.proposals.waitlisted.confirmed.count} (#{event.waitlisted_percent})
-
-      .widget
-        .widget-header
-          %i.fa.fa-tags
-          %h3 Proposal Tags
-        .widget-content
-          =event.valid_proposal_tags
-
-      .widget
-        .widget-header
-          %i.fa.fa-tags
-          %h3 Review Tags
-          = link_to "<i class='fa fa-pencil'></i> Edit".html_safe, event_staff_edit_path, class: "btn btn-sm btn-primary pull-right"
-        .widget-content
-          =event.valid_review_tags
-
-      .widget
-        .widget-header
-          %i.fa.fa-tags
-          %h3 Fields
-          = link_to "<i class='fa fa-plus'></i> Add Custom Field".html_safe, event_staff_custom_fields_path(event), class: "btn btn-primary btn-sm pull-right"
-        .widget-content
-          %p=event.fields
-          %h4 Custom Fields
-          %p=event.custom_fields.join(", ")
+            -# %li.list-group-item
+            -#   %strong.text-primary Confirmed:
+            -#   %span.label.label-info #{event.proposals.waitlisted.confirmed.count} (#{event.waitlisted_percent})
 
     .col-sm-4
-      .widget
+      .widget.widget-table.checklist
         .widget-header
-          %i.fa.fa-tags
-          %h3 Tracks
+          %i.fa.fa-check-square-o
+          %h3 Checklist
         .widget-content
-          -if event.track_count.present?
-            - event.track_count.sort_by{|k,v| v}.reverse.each_slice(8).to_a.each do |row|
-              %ul#columns.list-inline
-                -row.each do |name, count|
-                  %li
-                    .label.label-success
-                      = name
-                      = count
-          -else
-            %p No Tracks
+          %table.table.table-striped.table-bordered
+            %tbody
+              %tr
+                %td.text-primary
+                  %strong Event Url:
+                - if event.url.present?
+                  %td.set Set
+                - else
+                  %td.missing Missing
```

Comment:
I feel like for the entire checklist either the item name or the status (Missing / Set / None) should be a link to where I can add or edit those things. The UX is awkward because my mind goes "Oh yeah, the event url is missing, I should add that" but then .... where do I go? It doesn't feel easy to add those things.